src/widgets/Menu.js

```diff
@@ -125,9 +136,11 @@ export default class Menu extends createjs.Container {
 
   onFocus() {
     this._focusIndicator.visible = true;
+    this.underline.visible = true;
```

Comment:
The indicator for the accessKey shouldn't be toggled in focus/blur handlers since that would mean a keyboard user needs to first tab to it to find out what the accessKey is which means the accessKey is not functioning as a shortcut.

Reply:
removed